dockerfiles/examples/local/README.md

```diff
@@ -11,10 +11,31 @@ NB: The output of installing this chart, will contain a generated example with t
 
 Examples of use:
 
-*helm install localfile hpcc-localfile/*
-*helm install localfile hpcc-localfile/ --set storage.common.hostpath /home/me/hpcc-data*
+#### Docker desktop (using wsl2):
+
+  helm install localfile examples/local/hpcc-localfile --set common.hostpath=/run/desktop/mnt/host/c/wsldata/hpcc
+  helm install mycluster hpcc/ --set global.image.version=XXX -f examples/local/values-localfile.yaml
+
+#### Docker desktop (using osx):
+
+  helm install localfile examples/local/hpcc-localfile --set common.hostpath=/tmp/hpcc-data
+  helm install mycluster hpcc/ --set global.image.version=XXX -f examples/local/values-localfile.yaml
+
+#### Minikube:
+  When running under minikube the directory will need to be mounted with a command
+
+    minikube mount <host-directory>:<minikube-host-directory> --gid=1000 --uid=999
+
+  which needs to remaining running while the cluster is active.
+
+  E.g.
+
+  helm install localfile examples/local/hpcc-localfile --set common.hostpath=/mnt/hpccdata
+  minikube mount /home/gavin/hpccdata:/mnt/hpccdata --gid=1000 --uid=999
```

Comment:
shouldn't this minikube command be run before the helm install .. hpcc-localfile?

Reply:
it can be run before or after, but probably clearer before.  I'll move.